lib/presentation/molecules/youtube_player_molecule.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:youtube_player_flutter/youtube_player_flutter.dart';
+
+class YoutubePlayerMolecule extends StatelessWidget {
+  const YoutubePlayerMolecule({
+    required this.url,
+    super.key,
+    this.hideThumbnail = false,
+    this.showVideoProgressIndicator = true,
+    this.progressIndicatorColor = Colors.red,
+    this.playedColor = Colors.red,
+    this.handleColor = Colors.red,
```

Comment:
All colors should come from theme

Reply:
Do you want me to handle Light Mode and dark mode as well? Is a color scheme already being selected based on the mode?